### DISCUSSION TITLE
Informal meeting: Monday 28th 17:30 UTC

### DISCUSSION THREAD

**Discussion by olzama** [delph-in/docs#92] [Team Posts]
Dear all,

Some of us will meet on Monday November 28th, 17:30 UTC, to talk about delphin-related work that we are doing, like a couple of weeks ago.

Weiwei @wsun106 kindly agreed to host using her Zoom account:

https://cl-cam-ac-uk.zoom.us/j/92649124250?pwd=L2YzUE1pU3NHSmlsVDVnS203VmpNdz09

Zoom: 926 4912 4250
Password: 674288

See some of you there!

**Comment by olzama** [comment]
A reminder that this is today 17:30 UTC!

**Comment by arademaker** [comment]
I am so sorry for missing the meeting. I made a huge confusion with my agenda.

**Comment by olzama** [comment]
No worries, @arademaker ! As we hope this to become regular meetings, I think there is absolutely no expectation that everyone can make it every time! Hope to see you on the 12th of December!
